src/api/spec/features/beta/webui/maintenance_workflow_spec.rb

```diff
@@ -0,0 +1,119 @@
+require 'browser_helper'
+
+RSpec.describe 'MaintenanceWorkflow', js: true, vcr: false do
```

Comment:
This first commit with title _"Implement incident maintenance feature under beta"_ introduces a test with `vcr: false` and also adds cassettes. This doesn't make much sense.